docs/stellar-core_example.cfg

```diff
@@ -101,6 +101,11 @@ KNOWN_PEERS=[
 "core-testnet2.stellar.org", 
 "core-testnet3.stellar.org"]
 
+# BOOTSTRAP_DNS_SRV (single string) default is '_stellar-core._tcp.stellar.org'
```

Comment:
I guess my thinking here was to move increasingly towards "does something sensible with minimal configuration". If we had a default history source (stellar.org's public archive) as well, then you could literally run a nonvalidating stellar-core w/o any configuration.